reference/promotions.v3.yml

```diff
@@ -819,6 +819,39 @@ components:
                     minimum: 1
           required:
             - zone_ids
+    ShippingDiscountAction:
+      title: Shipping Discount Action
+      type: object
+      description: |-
+        **Shipping Discount Action**
+        Discounts shipping, optionally restricted to specific shipping methods.
+      properties:
+        shipping_discount:
+          type: object
+          properties:
+            discount:
+              $ref: '#/components/schemas/Discount'
+            as_total:
```

Comment:
I didn't notice this during initial UX review.

I'm wondering if this can be omitted to "per destination" for the 1st iteration ? distribute as a total amount is definitely doable engineering wise, but from past experience, it's also a source of rounding bugs, so we only want to take on the additional complexity when we're sure that the demand is there.

Probably worth asking JW to confirm if this is required for WD, if not, we can reduce the scope a bit.

Reply:
Agreed! Let's see if we can limit it to per desitination then expand with this later if needed.